ovgs.proto

```diff
@@ -272,34 +278,40 @@ service OwnershipVoucherService {
   // A user can only view roles of another user in the groups that
   // it has a role assigned to.
   // Errors will be returned:
+  // INVALID_ARGUMENT if the username is empty
   // NOT_FOUND if the group or the user doesn't exist.
-  // PERMISSION_DENIED if the user doesn't have access to the group
```

Comment:
What group was this referring to? This method would return, for the user specified in the message, the list of all roles and the groups over which it has that role. 

Also, `A user can only view roles of another user in the groups that it has a role assigned to.` This would mean that the calling user should only be able to see the group to role association for the specified user for groups it has some role on, directly or not? Right now, the implementation simply returns all the (role, group) pairs, with no filtering based on the callers privileges.